keep/api/routes/preset.py

```diff
@@ -62,7 +62,7 @@ def pull_alerts_from_providers(
             provider_class.get_alerts_by_fingerprint(tenant_id=tenant_id)
         )
         for fingerprint, alert in sorted_provider_alerts_by_fingerprint.items():
-            process_event(
+            process_event_sync(
```

Comment:
It didn't run process_event because of some async mess. I didn't manage to find the reason, but seems like this solution is the only one not giving flaky results.

get_running_loop and get_event_loop didn't work :/